system/audio.md

```diff
@@ -1,58 +1,70 @@
-2. Connection on vchan port 4714 used to transfer audio from Dom0/Audio VM to VM. Similar to previous one, audio data is sent as raw samples.
+1. Connection on vchan port 4713 used to transfer audio from VM to dom0/Audio VM.
+   There is no negotiation, no headers, etc.
+   Only raw samples are sent over this channel.
+2. Connection on vchan port 4714 used to transfer audio from dom0/Audio VM to VM.
+   Like the previous one, audio data is sent as raw samples.
 
-Additionally, the second vchan connection (on port 4714) is used in the opposite direction (VM->dom0) to send simple notifications/commands about audio state. Each such notification is a 4-bytes number in little-endian format.
+Additionally, the second vchan connection (on port 4714) is used in the opposite direction (VM to dom0) to send simple notifications/commands about the audio state.
+Each such notification is a 4-byte number in little-endian format.
 
 List of defined codes:
 
- - `0x00010001` - VM want to receive audio input (some process is listening); prior to this message, `pacat-simple-vchan` will not send any audio samples to the VM
- - `0x00010000` - VM do not want to receive audio input (no process is listening anymore); after this message, `pacat-simple-vchan` will not send any audio samples to the VM
- - `0x00020000` - VM do not want to send audio output; informational for dom0, to avoid buffer under runs (may affect pulseaudio calculated delays)
- - `0x00020001` - VM do want to send audio output
+ - `0x00010001` -- VM wants to receive audio input (some process is listening); prior to this message, `pacat-simple-vchan` will not send any audio samples to the VM.
+ - `0x00010000` -- VM does not want to receive audio input (no process is listening anymore); after this message, `pacat-simple-vchan` will not send any audio samples to the VM.
+ - `0x00020000` -- VM does not want to send audio output; informational for dom0, to avoid buffer under runs (may affect PulseAudio calculated delays).
+ - `0x00020001` -- VM does want to send audio output.
 
 pacat-simple-vchan
 ------------------
 
-Dom0 (or Audio VM) part of the audio virtualization. It is responsible for transferring audio samples between pulseaudio daemon in Dom0/Audio VM (having access to actual audio hardware) and a VM playing/recording.
-For each VM, matching `pacat-simple-vchan` process is running. Each of them open a separate input and output streams to local pulseaudio, which allows to control volume of each VM separately (using `pavucontrol` tool for example).
+This is the dom0 (or Audio VM) part of the audio virtualization.
+It is responsible for transferring audio samples between the PulseAudio daemon in dom0/Audio VM (which has access to the actual audio hardware) and a VM playing/recording sounds.
+In each VM, a corresponding `pacat-simple-vchan` process is running.
```

Comment:
`pacat-simple-vchan` is running in dom0, separate for each VM having audio. Not in a VM.

Reply:
Fixed, thanks.